tsconfig.json

```diff
@@ -5,17 +5,18 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "lib": ["es6", "es7"],
+    "skipLibCheck": true,
     "allowJs": true,
     "sourceMap": true,
-    "declaration": true,
+    "declaration": false,
     "strictPropertyInitialization": true,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
     "forceConsistentCasingInFileNames": true,
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
I think this is a good rule, do we have a lot of those? Can't we remove them if they are unused?

Reply:
There are 4 of them. They are just not used or marked as `TODO` by you. I can comment them out if you will.

Reply:
I'd prefer to have fixes in code that to disable the rule.